export/fortran/libint_f.F90

```diff
@@ -47,27 +65,27 @@ MODULE libint_f
 #endif
 
 #ifdef INCLUDE_ERI2
-   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_eri, 0:libint2_max_am_eri), &
+   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_2eri, 0:libint2_max_am_2eri), &
       BIND(C) :: libint2_build_2eri
 #if INCLUDE_ERI2 >= 1
-   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_eri1, 0:libint2_max_am_eri1), &
+   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_2eri1, 0:libint2_max_am_2eri1), &
       BIND(C) :: libint2_build_2eri1
 #endif
 #if INCLUDE_ERI2 >= 2
-   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_eri2, 0:libint2_max_am_eri2), &
+   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_2eri2, 0:libint2_max_am_2eri2), &
       BIND(C) :: libint2_build_2eri2
 #endif
 #endif
 
 #ifdef INCLUDE_ERI3
-   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am, 0:libint2_max_am, 0:libint2_max_am_3eri), &
+   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_default, 0:libint2_max_am_default, 0:libint2_max_am_3eri), &
       BIND(C) :: libint2_build_3eri
 #if INCLUDE_ERI3 >= 1
-   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am, 0:libint2_max_am, 0:libint2_max_am_3eri1), &
+   TYPE(C_FUNPTR), DIMENSION(0:libint2_max_am_default1, 0:libint2_max_am_default1, 0:libint2_max_am_3eri1), &
```

Comment:
I get a compilation error here because the macro `LIBINT2_MAX_AM_default1` is not defined.

Reply:
should be fixed now 🤞

Reply:
It is fixed indeed, thank you!